checkstyle.xml

```diff
@@ -82,7 +82,15 @@
     <module name="ParenPad"/>
     <module name="TypecastParenPad"/>
     <module name="WhitespaceAfter"/>
-    <module name="WhitespaceAround"/>
+    <module name="WhitespaceAround">
+      <property name="tokens"

```

Comment:
Our IntelliJ code style doesn’t put a space before the `}` in this annotation:

```
  @Option(name = {"-X", "--request"}, description = "Specify request command to use")
  public String method;
```

So here I manually remove `RCURLY` to make checkstyle happy. Sigh!